docs/user/cli-installation.rst

```diff
@@ -17,19 +17,53 @@ It is easiest to install the CLI using `homebrew <https://brew.sh/>`_:
   brew install renku
 
 
-Pip Script Installer (``pipsi``)
---------------------------------
+Isolated environments using ``pipx``
+------------------------------------
 
-You can use `pipsi <https://github.com/mitsuhiko/pipsi>`_ to isolate
-dependencies and to guarantee that there are no version conflicts. Make sure
-you have the ``pipsi`` command correctly installed and ``~/.local/bin`` is in
-your ``PATH``.
+Install and execute Renku in an isolated environment using ``pipx``.
+It will guarantee that there are no version conflicts with dependencies
+you are using for your work and research.
 
-.. code-block:: console
+.. note::
+
+  This is the method of installation in the renku docker images,
+  i.e. the default environment you use when you launch a JupyterLab session
+  via the renku browser interface.
+
+`Install pipx <https://github.com/pipxproject/pipx#install-pipx>`_
+and make sure that the ``$PATH`` is correctly configured.
+
+::
+
+    $ python3 -m pip install --user pipx
+    $ pipx ensurepath
+
+Once ``pipx`` is installed use following command to install ``renku``.
+
+::
+
+    $ pipx install renku
```

Comment:
(Following the docs) On alpine this command fails with:

```
/ # pipx install renku
Command "/root/.local/pipx/venvs/renku/bin/python -u -c "import setuptools, tokenize;__file__='/tmp/pip-install-zar2sap7/lxml/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-record-tdez0xs_/install-record.txt --single-version-externally-managed --compile --install-headers /root/.local/pipx/venvs/renku/include/site/python3.6/lxml" failed with error code 1 in /tmp/pip-install-zar2sap7/lxml/
'/root/.local/pipx/venvs/renku/bin/python -m pip install renku -q' failed
```

Should we add information about which binary packages are needed by renku?

Reply:
I think we need a section for troubleshooting and this can go there - can you make an issue for it?